packages/i18n/lib/prepare_build.ts

```diff
@@ -0,0 +1,19 @@
+import { cpSync, existsSync, mkdirSync } from 'node:fs';
+import { resolve } from 'node:path';
+import setRelatedLocaleImports from './set_related_locale_import.js';
+
+(() => {
+  // const isDev = process.env.__DEV__ === 'true';
+  const isDev = true;
```

Comment:
is it TODO?

Reply:
Yeah, it was placeholder till I'll merge #725 
That's why i've changed branch to `dev`

Reply:
How about adding a `//FIXME` or `//TODO` comment so you don't forget?

Reply:
I won't forget, i'll be my next work :)